app/utils/ControllerHelper.scala

```diff
@@ -37,6 +37,12 @@ import scala.concurrent.Future
 
 object ControllerHelper {
 
+  def hasIncompleteResponsiblePerson(rps: Option[Seq[ResponsiblePerson]]): Boolean = {
+    rps.map {
+      case data => ResponsiblePerson.filter(data).exists(_.isComplete == false)
```

Comment:
Yes, it could be a potential problem. I had a discussion with @kerryannekane to cover this possibly in the Acceptance Tests. I didnt want to prematurely assume its a problem

Reply:
the filter function actually does @ajchilts